index.js

```diff
@@ -84,9 +84,12 @@ app.on('ready', () => {
 		tray,
 		showOnAllWorkspaces: true,
 		preloadWindow: true,
-		showDockIcon: false,
+		showDockIcon: false, // no longer seems to work
 		icon: image,
 	});
+	if (app.dock) { // check if dock is available (macOS)
+	    app.dock.hide()
```

Comment:
not sure this works. seems a duplicate of https://github.com/smol-ai/menubar/blob/8a194f208524aedc4c6357183c31ed2c62129267/index.js#L98